components/configurator/ui/ConfiguratorRecaptcha.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+import GoogleReCAPTCHA from 'react-google-recaptcha'
```

Comment:
Non-blocker: why not reCaptcha v3 here?

Reply:
The v3 concept is pretty cool: https://www.youtube.com/watch?v=tbvxFW4UJdU, however, I'm not sure if we have a good amount of interactions to rely on it to come up with a score, so, maybe v2 should be fine here, as the only entry point, for now, is `/join`. We can try out https://github.com/t49tran/react-google-recaptcha-v3 to verify how different it would be, and, whether or not, it's a reliable one, which seems to be constantly updated.